kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -9902,7 +9902,7 @@ messages:
       return;
    }
 
-   BreakTrance(event = 0, what = $)
+   BreakTrance(event = 0, castwhat = $, usewhat = $)
```

Comment:
I had asked for a comment here explaining "usewhat", and instead you renamed "what" :-)  Can you please add a comment here explaining what the arguments mean?

Reply:
ok - will do - sorry for the confusion.  I think castwhat is better than just 'what' but I will add a comment or a doc-string.

Reply:
I added documentation in the form of a doc string - it explains all the arguments and what the function does.